tmt/steps/discover/fmf.py

```diff
@@ -430,30 +442,30 @@ def assert_git_url(plan_name: Optional[str] = None) -> None:
                             shutil.copytree(
                                 git_root, self.testdir, symlinks=True)
 
-            # Initialize or remove fmf root
-            if dist_git_init:
-                if not dist_git_extract:
-                    dist_git_extract = sourcedir
-                if not self.opt('dry'):
-                    fmf.Tree.init(dist_git_extract)
-            elif dist_git_remove_fmf_root:
-                if not self.opt('dry'):
-                    shutil.rmtree(os.path.join(
-                        dist_git_extract or top_fmf_root, '.fmf'))
-                if not dist_git_extract:
-                    dist_git_extract = sourcedir
-
-            # Now can safely default to top_fmf_root
+            # Set dist_git_extract if not set yet
             if not dist_git_extract:
-                dist_git_extract = top_fmf_root
+                dist_git_extract = [{
+                    "src": sourcedir,
+                    "dest": '.',
```

Comment:
Please, stick to `Path` as much as possible - `sourcedir` is probably already a `Path` instance, but instead of `'.'`, `Path.cwd()` would be much better. We try to force "path-like strings" to be represented as `Path` instances, for better control and linting.

Reply:
Oh indeed, I forgot `Path('.')`  does not expand itself. I need it in the form of `.` because I use it to `join_path` later on.